docs/src/content/docs/api/voices.mdx

```diff
@@ -67,8 +67,77 @@ Lists all available voices. Currently only two voices are available: `lily` and
   </TabItem>
 </Tabs>
 
-### Example: List All Available Voices
-<Tabs syncKey="constellations">
+## Create (Clone) Voice
+
+<Badge text="POST /voices" variant="success" size="large" />
+
+Create a new cloned voice. API key scoped.
```

Comment:
Still not sure what "API key scoped" means, but I think it's worth saying that the voice is private to your account.  (Maybe that's what you mean? It's not true that it's restricted to your API key though since an account can have multiple API keys.)

Reply:
yes that is what i mean. so then are all things account scoped and not api key scoped?

Reply:
All resources are either public or owned by an Account.